app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -68,8 +68,8 @@ def create
 
   def edit
     return if @submission.attempting?
-
-    render 'blocked' if @submission.assessment.block_student_viewing_after_submitted? && current_user.normal?
+    current_course_user = @course.course_users.for_user(current_user).first
```

Comment:
Is there a more direct way to query for the course user? Not sure, just checking to be sure.

Reply:
Yeah I was looking for a more direct way to query this as well but current user is not linked to course_user so need to go query through the course user table.

Reply:
Randomly popping in but there is such a method - https://github.com/Coursemology/coursemology2/blob/master/app/controllers/course/controller.rb#L32-L42

Reply:
This is nice! thanks!